core/api/service/state/requests/get_storage_size.hpp

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright Quadrivium LLC
+ * All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <jsonrpc-lean/request.h>
+
+#include <optional>
+
+#include "api/service/state/state_api.hpp"
+
+#include "outcome/outcome.hpp"
```

Comment:
Following convention , base classes headers emplace first, then SLT headers, then 3rd party libs headers, and last ones which needed for project. Each such group separated by empty line, and lexicografical sorted.